2203/2/versus.rb

```diff
@@ -0,0 +1,112 @@
+require 'optparse'
+require 'terminal-table'
+require 'russian_obscenity'
+require './fix_words'
+
+# This class can show some intresting stats about russian battle rapers
+class Versus
```

Comment:
what is the sense of this class? it looks like wrapper for your non OOP solution.